### PR TITLE
test: Support booting vanilla VMs

### PR DESCRIPTION
**Pull request by lmars**
This will be useful for testing installation of Flynn components.

### CODE REVIEW

**Comment by titanous** [comment]
LGTM

File: test/runner/runner.go [hunk 1]
```diff
@@ -165,6 +165,7 @@ func (r *Runner) start() error {
 	router.ServeFiles("/assets/*filepath", http.Dir(args.AssetsDir))
 	router.GET("/cluster/:key/:cluster", r.clusterAPI(r.getCluster))
 	router.POST("/cluster/:key/:cluster", r.clusterAPI(r.addHost))
+	router.POST("/cluster/:key/:cluster/vanilla", r.clusterAPI(r.addVanillaHost))
```
Comment (heavenlyhash) [inline test/runner/runner.go]: This seems more like it should be a parameter or part of the post body to me than a separate endpoint.

**Comment by lmars** [comment]
@heavenlyhash comments addressed

**Comment by heavenlyhash** [comment]
LGTM

**Pull request merged**